convert2rhel/main.py

```diff
@@ -202,7 +222,13 @@ def _raise_for_skipped_failures(results):
     failures = actions.find_actions_of_severity(results, "SKIP", level_for_raw_action_data)
     if failures:
         # The report will be handled in the error handler, after rollback.
-        loggerinst.critical("Conversion failed.")
+        method = "conversion" if toolopts.tool_opts.activity == "conversion" else "analysis"
+        message = (
+            "The {method} process failed.\n\n"
+            "A problem was encountered during {method} and a rollback will be "
+            "initiated to restore the system as the previous state."
+        ).format(method=method)
+        raise _InhibitorsFound(message)
```

Comment:
Can we use activity directly instead of this inline if statement?

Reply:
Good point! We totally can: 

```suggestion
        message = (
            "The {method} process failed.\n\n"
            "A problem was encountered during {method} and a rollback will be "
            "initiated to restore the system as the previous state."
        ).format(method=toolopts.tool_opts.activity.title())
        raise _InhibitorsFound(message)
```

Reply:
something like this?

Reply:
Updated the code. The `title()` part is not needed.